src/RF24LogHandler.h

```diff
@@ -38,10 +38,11 @@ class RF24LogHandler
      * same order for which they appear in the @p message
      */
     virtual void log(uint8_t logLevel,
-                     const __FlashStringHelper *vendorId,
+                     const char *vendorId,

```

Comment:
This method is available for all architectures. However we could exclude it on AVR.
All log handlers are updated as well.